react/features/reactions/actions.any.ts

```diff
@@ -56,12 +49,8 @@ export function sendReactions() {
  * Adds a reaction to the local buffer.
  *
  * @param {string} reaction - The reaction to be added.
- * @returns {{
- *     type: ADD_REACTION_BUFFER,
- *     reaction: string
- * }}
  */
-export function addReactionToBuffer(reaction: string) {
+export function addReactionToBuffer(reaction: string): Object {
```

Comment:
I would extend the ReactionsAction interface to optionally accept `reaction`, `reactions` and `message` (or if it's not too much work, refactor both as `payload`) to use it as a return type for all these methods.